contentcuration/contentcuration/frontend/channelEdit/views/ResourcePanel.vue

```diff
@@ -472,23 +472,28 @@
       /* VALIDATION */
       // License isn't specified
       noLicense() {
-        return !this.isTopic && validateNodeLicense(this.node).length;
+        return Boolean(!this.isTopic && validateNodeLicense(this.node).length);
       },
       // Copyright holder isn't set on non-public domain licenses
       noCopyrightHolder() {
-        return !this.isTopic && !validateNodeCopyrightHolder(this.node).length;
+        return Boolean(!this.isTopic && validateNodeCopyrightHolder(this.node).length);
       },
       // License description isn't provided on special permissions licenses
       noLicenseDescription() {
-        return !this.isTopic && !validateNodeLicenseDescription(this.node).length;
+        return Boolean(
+          this.license &&
+            this.license.is_custom &&
```

Comment:
I am not sure if I follow, `is_custom` check is actually part of `validateNodeLicenseDescription` [logic](https://github.com/learningequality/studio/blob/develop/contentcuration/contentcuration/frontend/shared/utils/validation.js#L165).

What's the criteria actually? After we're clear on that, can we make it so it's defined only in `validateNodeLicenseDescription`? The idea behind `shared/utils/validation` is to have all validation criteria defined at one place.

Reply:
On the same note, it might be good to remove `isTopic` checks etc. and move them to validators functions too (they receive a node as a parameter, so the information is available). I didn't realize this when working on validation - I can do it as a follow-up.

Reply:
I'm going to need to take some time to get back into this later on. Adding the `is_custom` check here made it work properly I thought.

I'll test it - but seems like there may be a bug elsewhere.

Reply:
I went back in and I couldn't replicate my initial motivation for doing things this way. I'm wondering if maybe I'd corrupted data and hadn't realized it somehow?

I think there is something off just the slightest bit with the validations but I cannot put my finger on it. I'll follow up in a separate issue if I find anything in further work.